src/functions/tarsUpload/framework/di/inversify.config.ts

```diff
@@ -28,23 +28,49 @@ import { ISubmissionOutcomeUploader } from '../../application/secondary/ISubmiss
 import { HTTPSubmissionOutcomeUploader } from '../adapter/report/HTTPSubmissionOutcomeUploader';
 import { IOutcomeReportingHTTPConfig } from '../adapter/report/IOutcomeReportingHTTPConfig';
 import { EnvvarOutcomeReportingHTTPConfig } from '../adapter/report/EnvvarOutcomeReportingHTTPConfig';
-import { ITestResultHTTPConfig } from '../adapter/upload/ITestResultHTTPConfig';
-import { EnvvarTestResultHTTPConfig } from '../adapter/upload/EnvvarTestResultHTTPConfig';
+import { ITestResultHTTPConfig } from '../adapter/fetch/ITestResultHTTPConfig';
+import { EnvvarTestResultHTTPConfig } from '../adapter/fetch/EnvvarTestResultHTTPConfig';
 import { ITARSRateLimiterConfig } from '../adapter/upload/ITARSRateLimiterConfig';
 import { TARSRateLimiterConfig } from '../adapter/upload/TARSRateLimiterConfig';
 import { ILogger } from '../../domain/util/ILogger';
 import { ConsoleLogger } from '../../domain/util/ConsoleLogger';
+import { SuccessfulStubbingTARSUploader } from '../adapter/upload/SuccessfulStubbingTARSUploader';
+import { ConfigurableBatchFetcher } from '../adapter/fetch/ConfigurableBatchFetcher';
+import {
+  SuccessfulStubbingSubmissionOutcomeUploader,
+} from '../adapter/report/SuccessfulStubbingSubmissionOutcomeUploader';
+
+const deriveTARSUploaderType = () => {
+  if (process.env.USE_TARS_STUB === 'true') {
+    return SuccessfulStubbingTARSUploader;
+  }
+  return HTTPTARSUploader;
```

Comment:
Just something to consider going forward: when there are acronyms in class/function/variable names the convention is to use camel casing. So for example `HTTPTARSUploader` would be `HttpTarsUploader` - easier to read.

Reply:
Depends which convention we follow, it's different language-to-language, framework-to-framework, project-to-project. If we want to use strict camel-casing everywhere for DES, I will adopt this going forward.